### PR TITLE
Fix cli output, reboot and timeout

### PR DESCRIPTION
**Pull request by haslinghuis**
- [x] Fixes cli where MSP request characters became visible in terminal or file output. See: https://github.com/betaflight/betaflight/pull/11764
- [x] Fixes `rememberLastTab` when off configurator still returned to last used tab.
- [x] Resets MSP timeout upon opening on each connection.
- [x] Fix sonar issues: updated deprecated `navigator.appVersion`

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by daleckystepan — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
